src/main/java/com/dev/cinema/dao/CinemaHallDao.java

```diff
@@ -7,4 +7,6 @@
     CinemaHall add(CinemaHall cinemaHall);
 
     List<CinemaHall> getAll();
+
+    CinemaHall getById(Long id);
```

Comment:
let' use consistent naming: getById or findById(i would suggest  just get()) and use it everywhere